### ISSUE TITLE
BlueSCSI v1 firmware update needs a step-by-step set of instructions

### ISSUE THREAD

**Issue by someyob** [erichelgeson/BlueSCSI#264]
1.  How to use QMK for this?
2.  Which firmware, there's several to choose from?
3.  Am I plugging in to the Bluepill, or using a ST-Link?
4.  Do I need to change the jumpers on the Bluepill?

**Comment by tiff-lee** [comment]
Did you figure this out in the end? I took a bunch of pictures showing the process using the ST-Link and the ST32 programming software for the blue pills but I hadn't got around to knocking a guide up yet. If you still need it happy to do something.

**Comment by someyob** [comment]
Yes, I put it aside, and haven't gotten back to it.  So, yes please.  I would appreciate it, and I think the community would benefit.
Thank you for getting back to me.

**Comment by tiff-lee** [comment]
Ok sorry for the delay, i've just put something together its a little rough around the edges but pretty much anyone should be able to follow this even with little prior knowledge.
Note that this is using a clone ST-LINK V2 and the STM32 ST-LINK utility to flash the bluepill for bluescsi v1.1.

Not actually sure if this is worth integrating into the wiki somewhere, i'm not even sure how you go about doing that (don't use github enough).

[Flashing Guide.pdf](https://github.com/user-attachments/files/16411665/Flashing.Guide.pdf)

**Comment by someyob** [comment]
Lucky day, today was my day to do mac stuff.

I ran through the procedure, no issues whatsoever.  Only one ambiguous section:  When connecting the jumpers between the ST-Link and the BluePill, I chose to do it _without_ the ST-Link connected to the USB.   Once finished connecting, I plugged in the ST-Link to USB and carried on per page 3.  I figured it was safest to do it unpowered; maybe it's irrelevant.

Thanks very much for doing this, all booting fine.

**Comment by tiff-lee** [comment]
Excellent, yeah that text was pulled directly from the wiki there is no real danger either way. Main thing is you got it sorted which is good.

**Comment by talljon** [comment]
tiff-lee your Flashing Guide.pdf is very well done and easy to follow.  

Thank you for posting.
